x-pack/plugins/search_playground/public/components/sources_panel/sources_panel_for_start_chat.tsx

```diff
@@ -67,6 +82,19 @@ export const SourcesPanelForStartChat: React.FC = () => {
         </EuiFlexItem>
       )}
 
+      {sourceFieldErrorMessage && (
+        <EuiCallOut color="warning" iconType="warning">
+          <p>
+            {i18n.translate('xpack.searchPlayground.emptyPrompts.sources.warningCallout', {
+              defaultMessage: '{errorMessage}',
+              values: {
+                errorMessage: sourceFieldErrorMessage,
+              },
+            })}
```

Comment:
:x: This translate is not needed if we're just doing a pass through of the `sourceFieldErrorMessage`

```suggestion
            {sourceFieldErrorMessage}
```

Reply:
The translations were added considering that this error message will show up in the UI. My understanding was anything that displays to the user will need `translation`.

Reply:
Otherwise same key may responsible for different texts. In transform message we can initiate i18n for each text of error response if user might see it

Reply:
Got it, I added this while I am dealing with static text however, it is not static anymore.

Reply:
If you were adding other copy with the error then you would add a translation, something like:

```
 {i18n.translate('xpack.searchPlayground.emptyPrompts.sources.warningCallout', {
            defaultMessage: 'Warning chat copy: {errorMessage}',
            values: {
              errorMessage: sourceFieldErrorMessage,
            },
            })}
```

But if the message is just a variable then there is no reason to have a translation.

Reply:
updated as suggested

Reply:
cc: @leemthompo for the copy